### PR TITLE
Added github image to folder, created HTML file

### PR DESCRIPTION
**Pull request by Stephenich**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -16,11 +16,31 @@
...
+  <h1><a class="name" href="https://github.com/Stephenich/stephenich.github.io">Stehen Gurney</a></h1>
+    <img src="images/githubIcon.svg" alt="GitHub Logo">
+  </div>
+   <header> <div class="menu">
+      <ul>
+        <li>Home</li>
+        <li>GitHub</li>
+      </ul>
+    </div>
+  </header>
```
Comment (roughcoder) [inline src/index.html]: a better semantic layout would be something like this 

```
<header>
    <div className="header">
        <h1>
            <a
                className="name"
                href="https://github.com/Stephenich/stephenich.github.io">
                Stehen Gurney
            </a>
        </h1>
        <img
            src="images/githubIcon.svg"
            alt="GitHub Logo"
        />
    </div>
    <nav className="menu">
        <ul>
            <li>Home</li>
            <li>GitHub</li>
        </ul>
    </nav>
</header>
```

File: src/index.html [hunk 2]
```diff
@@ -16,11 +16,31 @@
...
 
-  <p>Hello world! This is my HTML will this work??.</p>
-  <img src="images/bigben.jpg" alt="bigben" width="400" height="400"> 
-  <img src="images/london.jpg" alt="london" width="400" height="400">
+  <div>
```
Comment (roughcoder) [inline src/index.html]: you could use `<article>`  or even `<main>`

File: src/index.html (hunk 1 above)
Comment (Stephenich) [reply src/index.html]: I'll be sure to change it to be more semantic and readable.

**Comment by roughcoder** [comment]
also get some kind of html formatting added to your vscode on save

File: src/index.html (hunk 2 above)
Comment (Stephenich) [reply src/index.html]: I'll look into the difference of these tags, and change it accordingly

**Review by roughcoder — APPROVED** [review]

**Pull request merged**